### PR TITLE
Shooter

### PR DESCRIPTION
**Pull request by zacharyCormack**
# ADDED SHOOTER MATH

added a shooter math function, which determines power to distance.
means driver need not intuitively understand fixed angle aim system.
also added empty joystick elements for shooter, as those were missing.

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -35,252 +31,222 @@
...
+
+		shooterSubsystem.getMonitorGroup().setOverheatWarningCallback((motor, temp) -> {
+			if (!shooterSubsystem.getOverheatShutoffOverride())
+				shooterMotorStatus.withProperties(Map.of("color when false", Constants.COLOR_MOTOR_WARNING)).getEntry().setBoolean(false); 
+			getLogger().logWarning("Shooter motor " + motor.getDeviceId() + " reached overheat warning at " + temp + " C!"); 
+		});
+
+	}
+
+	private double shooterDistToSpeed(double dist, double endh) {
```
Comment (TeamArctos6135) [inline src/main/java/frc/robot/RobotContainer.java]: If this is a shooter-specific function, I think we should move this to the commands or subsystems folder as RobotContainer.java usually does not include any subsystem-specific functions.

**Review by TeamArctos6135 — CHANGES_REQUESTED** [review]
To merge into the master branch, we should test the code on the robot (which has not been built yet). For now, I'll merge only the drivetrain into master and keep this on the shooter branch. Sound good?

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (zacharyCormack) [reply src/main/java/frc/robot/RobotContainer.java]: Ok, that's probably a good idea.

Comment (zacharyCormack) [reply src/main/java/frc/robot/RobotContainer.java]: I did that

**Pull request closed without merging**